packages/stylelint-config-nikita/index.js

```diff
@@ -132,5 +132,19 @@ module.exports = {
         "value-list-comma-space-after": "always-single-line",
         "value-list-comma-space-before": "never",
         "value-list-max-empty-lines": 0,
+        
+        "value-no-vendor-prefix": true,
+        "property-no-vendor-prefix": true,
+        "at-rule-no-vendor-prefix": true,
+        "declaration-no-important": true,
+        "color-named": "never",
+        "font-weight-notation": "numeric",
+        "max-nesting-depth": 3,
```

Comment:
I think this is a little too strict here... what about 5?

Reply:
I thought so too but our guidelines say:
> Maximum Nesting: three levels deep!
But maybe 5 is more practical. But then we have to also update the README, right?